### DISCUSSION TITLE
Question about the Distributed Deployment Architecture

### DISCUSSION THREAD

**Discussion by damienhull** [Security-Onion-Solutions/securityonion#4464] [Unsupported Versions]
Here's what I would like to do. 

1. Monitor network traffic with forward nodes
2. Monitor laptops with Wazuh
3. Manager server in the cloud
4. Laptop Wazuh data needs to be sent directly to the cloud
5. Laptops won't always be in the office 

Is this possible? 

Thanks!

**Comment by weslambert** [comment]
Yes 😃
